### PR TITLE
Add quiet flag to up

### LINKED ISSUE #386: Allow way to turn off progress meter
It would be nice to have a flag that disabled the progress meter during `draft up`. This is because in some non-interactive presentations of STDOUT, they ignore the backspace character, which means I get a bazillion lines of output.

Maybe just `draft up --quiet`, which would just display errors, and nothing else?

### PR DESCRIPTION
**Pull request by sj14**
This PR adds the `--quiet` flag to the `up` command (fixes #386).

### CODE REVIEW

File: cmd/draft/up.go [hunk 1]
```diff
@@ -118,10 +120,11 @@ func newUpCmd(out io.Writer) *cobra.Command {
 	f.BoolVar(&up.dockerClientOptions.Common.Debug, "docker-debug", false, "Enable debug mode")
 	f.StringVar(&up.dockerClientOptions.Common.LogLevel, "docker-log-level", "info", `Set the logging level ("debug"|"info"|"warn"|"error"|"fatal")`)
 	f.BoolVar(&up.dockerClientOptions.Common.TLS, "docker-tls", defaultDockerTLS(), "Use TLS; implied by --tlsverify")
-	f.BoolVar(&up.dockerClientOptions.Common.TLSVerify, fmt.Sprintf("docker-%s", dockerflags.FlagTLSVerify), defaultDockerTLSVerify(), "Use TLS and verify the remote")
+	f.BoolVar(&up.dockerClientOptions.Common.TLSVerify, fmt.Sprintf("docker-%s", dockerflags.FlagTLSVerify), defaultDockerTLSVerify(), "use TLS and verify the remote")
```
Comment (radu-matei) [inline cmd/draft/up.go]: Nice catch, thanks!

File: cmd/draft/up.go [hunk 2]
```diff
@@ -118,10 +120,11 @@ func newUpCmd(out io.Writer) *cobra.Command {
...
 	f.StringVar(&up.dockerClientOptions.ConfigDir, "docker-config", cliconfig.Dir(), "Location of client config files")
-	f.BoolVarP(&autoConnect, "auto-connect", "", false, "specifies if draft up should automatically connect to the application")
-	f.BoolVar(&skipImagePush, "skip-image-push", false, "skip pushing image to registry")
+	f.BoolVarP(&autoConnect, "auto-connect", "", false, "Specifies if draft up should automatically connect to the application")
```
[the remark this answers is not in the document]
Comment (sj14) [reply cmd/draft/up.go]: Ah, I just saw the inconsistency of uppercase and lowercase letters. First, I couldn't decide which style to use and now it's still mixed, as I see 😁 
I can change them all to lowercase or make a new MR, however you like. I hope this was the aim of your question.

[the remark this answers is not in the document]
Comment (radu-matei) [reply cmd/draft/up.go]: Yeah, as across Draft we have all descriptions starting with lowercase letters, we should enforce that  practice 😄

[the remark this answers is not in the document]
Comment (sj14) [reply cmd/draft/up.go]: All descriptions should now start with lowercase letters, please check again :)

**Pull request merged**